### ISSUE TITLE
Convert Sass variables to CSS custom properties

### ISSUE THREAD

**Issue by filoxo** [filoxo/solid-state-react#13]
One of the objectives of this project is to move away from the complicated Sass setup into something a little more modern. I'd like to begin by moving variables from [_vars.scss](https://github.com/filoxo/solid-state-react/blob/master/assets/sass/libs/_vars.scss) (only the ones that make sense, which would be everything except for font variables) into custom properties in index.css, with the goal of reusing them throughout the individual components.

**Comment by TremaineNeethling** [comment]
I'm working on this.

**Comment by TremaineNeethling** [comment]
Hey @filoxo 

Below is how the palette is currently defined in sass

```
$palette: (
	bg:						#2e3141,
	fg:						#ffffff,
	fg-bold:				#ffffff,
	fg-light:				rgba(255,255,255,0.35),
	border:					rgba(255,255,255,0.125),
	border-bg:				rgba(255,255,255,0.025),
	border2:				rgba(255,255,255,0.25),
	border2-bg:				rgba(255,255,255,0.075),
	accent:					#4c5c96
);
```

Is the below conversion what you had in mind?

```
--palette-bg: #2e3141;
--palette-fg: #ffffff;
--palette-fg-bold: #ffffff;
--palette-fg-light: rgba(255,255,255,0.35);
--palette-border: rgba(255,255,255,0.125);
--palette-border-bg: rgba(255,255,255,0.025);
--palette-border2: rgba(255,255,255,0.25);
--palette-border2-bg: rgba(255,255,255,0.075);
--palette-accent: #4c5c96;
```

**Comment by filoxo** [comment]
Yea, something like that! I don't think the `palette` prefix is necessary or helpful, so if you could drop that; and then add those styles to index.css then I can begin converting the component styles that use them. I look forward to your PR! Thanks so much!

**Comment by TremaineNeethling** [comment]
How does this look for you?
```
--border-radius: 5px;
--element-height: 2.75em;
--element-margin: 2em;
--inner-size: 55em;
--section-spacing-large: 3em;
--section-spacing-medium: 2em;
--section-spacing-small: 1.75em;
--wrapper-edges-large: 6.5em;
--wrapper-edges-medium: 4.75em;
--wrapper-edges-small: 2.5em;

--font-family: 'Source Sans Pro', Helvetica, sans-serif;
--family-fixed: 'Courier New', monospace;
--family-heading: Raleway, Helvetica, sans-serif;
--weight: 300;
--weight-bold: 600;
--weight-heading: 200;
--weight-heading-bold: 700;
--kern-heading: 0.1em;

--bg: #2e3141;
--fg: #fff;
--fg-bold: #fff;
--fg-light: rgba(255,255,255,0.35);
--border: rgba(255,255,255,0.125);
--border-bg: rgba(255,255,255,0.025);
--border2: rgba(255,255,255,0.25);
--border2-bg: rgba(255,255,255,0.075);
--accent: #4c5c96;
```

**Comment by filoxo** [comment]
Yep, that looks great to me!

**Comment by filoxo** [comment]
Resolved in #14
